acquia/cli
# GL-3134: Add new Gitlab variable for Node projects.

**Motivation**

Fixes [GL-3134](https://acquia.atlassian.net/browse/GL-3134)

**Proposed changes**

New Node.js projects created in Code Studio pass a new variable of `NODE_HOSTING_TYPE` with a default value of `basic`. When Node V3 launches, we will update the user prompts to ask for a Node hosting type.

**Alternatives considered**

N/A

**Testing steps**


1. Follow the [contribution guide](https://github.com/acquia/cli/blob/HEAD/CONTRIBUTING.md#building-and-testing) to set up your development environment or [download a pre-built acli.phar](https://github.com/acquia/cli/blob/HEAD/CONTRIBUTING.md#automatic-dev-builds) for this PR.
2. Clear the kernel cache to pick up new and changed commands: `./bin/acli ckc`
3. Run `acli cs:wizard --gitlab-host-name=code.dev.cloudservices.acquia.io` (or other env as needed)
4. Select prompts for a Node.js application
5. Confirm you do not see a prompt for Node hosting type
6. When the project is created in Gitlab, an environment variable should exist for `NODE_HOSTING_TYPE` with a value of `basic`. Variable should not be protected or masked.
